src/Title.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+
+export default function Title(props) {
```

Comment:
This is a good component :) it's also possible (and more conventional) to specify the title in the `<head>` element of your `index.html`.